tests/syntax-tests/create_highlighted_versions.py

```diff
@@ -64,6 +64,9 @@ def create_highlighted_versions(output_basepath):
                 file=sys.stderr,
             )
             sys.exit(1)
+        except FileNotFoundError:
+            print("'bat' was not found on path.")
```

Comment:
Maybe:
```
Error: Could not execute 'bat'. Please make sure that the executable is available on the PATH.
```

Reply:
Thank you. This sounds so much better :)
Updated!